### DISCUSSION TITLE
Admin dashboard show-hide the vertical "Navigation" bar

### DISCUSSION THREAD

**Discussion by kanenas** [opencart/opencart#12853] [Q&A] [answered]
Is there any modification for Opencart admin dashboard to **show-hide** the vertical "Navigation" bar?
I mean like it was until Opencart **v2.3.0.2**. Working on a laptop makes it difficult to operate!
![opencart-admin-dashboard-show-hide-navigation](https://github.com/opencart/opencart/assets/327786/bc0c0366-8226-437f-aef9-00459e57e839)

**Comment by condor2** [comment, marked as answer]
There is one for 3.0.x.x version

https://www.opencart.com/index.php?route=marketplace/extension/info&extension_id=42126
